config/webpack.config.js

```diff
@@ -171,17 +153,38 @@ const configGenerator = (options) => {
       noParse: [/mapbox\/vendor\/promise.js$/],
     },
     resolve: {
-      alias: {
-        modernizr$: path.resolve(__dirname, './modernizrrc')
-      },
       extensions: ['.js', '.jsx']
     },
+    optimization: {
```

Comment:
This is the v4 replacement for adding commons chunk and uglify to the plugins list.